mods/parsoid.js

```diff
@@ -746,6 +746,10 @@ PSP.makeTransform = function(from, to) {
             // Handle body_only flag.
             // bodyOnly is deprecated and will be removed at some point
             if (to === 'html' && (req.body.body_only || req.body.bodyOnly)) {
+                // Log remaining bodyOnly uses / users
+                if (req.body.bodyOnly) {
```

Comment:
Could you also please add a TODO / FIXME / XXX / TEMP comment here so that we remember to remove it ultimately?